src/java.base/share/classes/java/lang/String.java

```diff
@@ -698,7 +698,7 @@ private String(Charset charset, byte[] bytes, int offset, int length) {
     /*
      * Throws iae, instead of replacing, if malformed or unmappable.
      */
-    static String newStringUTF8NoRepl(byte[] bytes, int offset, int length) {
+    static String newStringUTF8NoRepl(byte[] bytes, int offset, int length, boolean noCopy) {
```

Comment:
The parameter will be easier to understand if `true` means that the string created must not share the byte array supplied.
For example,

\@param noShare {@code true} if the resulting string MUST NOT share the byte array, {@code false} if the byte array can be exclusively used to construct the string and is not modified or used for any other purpose.

This flips the boolean argument in the uses below.

Reply:
> The parameter will be easier to understand if `true` means that the string created must not share the byte array supplied. For example,
> 
> @param noShare {@code true} if the resulting string MUST NOT share the byte array, {@code false} if the byte array can be exclusively used to construct the string and is not modified or used for any other purpose.
> 
> This flips the boolean argument in the uses below.

I modified it according to your suggestion.